### ISSUE TITLE
Wrong bounding box? when using the color black in an animation

### ISSUE THREAD

**Issue by oli4warin** [LuCEresearchlab/pytamaro#40]
In certain constellations there seems to be a problem with the color black in show_animation:
As in the example below, the large black square in the second frame is not fully displayed. 
If you change the color from black to red, everything works as desired.
```
from pytamaro import (
    overlay,
    ellipse, rectangle, 
    red, green, black, 
    show_animation
)

show_animation(
    [
        rectangle(120,120,black),
        overlay(ellipse(10,10,green),rectangle(120,120,black))
    ], 
    duration=1000
)
```
It might be related to the default background color in generating the gif.

![image](https://github.com/user-attachments/assets/c41bec47-c61a-43a0-8312-21ebf7297681)
![image](https://github.com/user-attachments/assets/96206398-2af8-444a-923a-783fd9c613c1)

**Comment by lucach** [comment]
Hi @oli4warin, thanks for reporting this!

We bumped into this weird behavior a while ago (internal issue: https://github.com/LuCEresearchlab/pytamaro-web/issues/242). It turns out to be a bug in the optimizer of Pillow's GIF encoder. [We reported the issue upstream and it has already been fixed](https://github.com/python-pillow/Pillow/issues/8222). The bugfix will be part of [Pillow 11, which is scheduled to be released on October 15](https://github.com/python-pillow/Pillow/issues/8215).

I will release a new version of PyTamaro as soon as the new version of Pillow is released. I'm keeping this issue open to track this process.
